com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeReferenceVolumeAuthoringEditor.cs

```diff
@@ -165,12 +198,141 @@ private void Constrain()
             m_DilationValidityThreshold.floatValue = 1f - DilationValidityThresholdInverted;
         }
 
+        private void CheckInit()
+        {
+            if (debugMesh == null || debugMaterial == null)
+            {
+                // Load debug mesh, material
+                debugMesh = AssetDatabase.LoadAssetAtPath<Mesh>("Packages/com.unity.render-pipelines.core/Editor/Resources/DebugProbe.fbx");
+                UpdateDebugMaterial();
+            }
+        }
+
+        void UpdateDebugMaterial()
+        {
+            Shader debugShader = Shader.Find("Hidden/InstancedProbeShader");
+            if (GraphicsSettings.renderPipelineAsset is IOverrideCoreEditorResources overrideResources)
+                debugShader = overrideResources.GetProbeVolumeProbeShader();
+
+            debugMaterial = new Material(debugShader) { enableInstancing = true };
+        }
+
         public void OnSceneGUI()
         {
-            ProbeReferenceVolumeAuthoring pvra = target as ProbeReferenceVolumeAuthoring;
+            // if (Event.current.type == EventType.Repaint)
```

Comment:
Why this commented out line? leftover or reminder? If the latter a comment would be nice

Reply:
It was not supposed to be pushed, I'll revert it